analytics/policy/services/registration_number_policies.py

```diff
@@ -0,0 +1,39 @@
+import re
+import json
+import urllib3
+
+class RegistrationNumberPolicies:
+    def __init__(self, registration_number):
+        self.registration_number = registration_number
+        self.session = urllib3.PoolManager()
+
+    def valid_pattern(self):
+        valid_registration_pattern = re.compile(r'^[0-9]{2}-[0-9]{6}$')
```

Comment:
The regex should check first two digits against year end digits.
Please check issue description